IDea: footer support like was with components (#48)
arnaudsm/raito: docs/footer.md

> 
> -----------------------------------------------------------------------------------
> 
> This was made with love by 💼 [https://arnaud.at](https://arnaud.at) under MIT License

Reviewer: ```suggestion
Made with love by [arnaudsm](https://arnaud.at) under MIT License
```
